src/MediawikiApi.php

```diff
@@ -66,21 +68,32 @@ public static function newFromApiEndpoint( $apiEndpoint ) {
 	}
 
 	/**
+	 * Create a new MediawikiApi object from a URL to any page in a MediaWiki website.
+	 *
 	 * @since 2.0
+	 * @see https://en.wikipedia.org/wiki/Really_Simple_Discovery
 	 *
 	 * @param string $url e.g. https://en.wikipedia.org OR https://de.wikipedia.org/wiki/Berlin
-	 *
 	 * @return self returns a MediawikiApi instance using the apiEndpoint provided by the RSD
 	 *              file accessible on all Mediawiki pages
-	 *
-	 * @see https://en.wikipedia.org/wiki/Really_Simple_Discovery
+	 * @throws UsageException If the RSD URL could not be found in the page's HTML.
```

Comment:
I think this is ready to go once this comment is fixed :)
Then a new release of mediawiki-api-base soon!